### ISSUE TITLE
Bug: ResizeObserverEntry.borderBoxSize and .contentBoxSize have wrong types

### ISSUE THREAD

**Issue by sven42** [tulz-app/laminext#190]
Hello,

thanks a lot for your nice laminar extension!

I found a small bug in your type declaration of `io.laminext.domext.ResizeObserverEntry`:

```
@js.native
trait ResizeObserverEntry extends js.Any {

  val borderBoxSize: ResizeObserverBorderBoxSize
  val contentBoxSize: ResizeObserverContentBoxSize
  val contentRect: DOMRect
  val target: HTMLElement | SVGElement
}
```

According to https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry borderBoxSize and contentBoxSize should have the type `js.Array[ResizeObserverBorderBoxSize]` and `js.Array[ResizeObserverContentBoxSize]`

In addition the Any-Types in e.g. ResizeObserverBorderBoxSize seem to be of type Double:

```
@js.native
trait ResizeObserverBorderBoxSize extends js.Any {

  val blockSize: Any
  val inlineSize: Any
}
```

The following example shows my current workaround:

```
resizeObserver --> { (entry: ResizeObserverEntry) =>
  try {
    entry.borderBoxSize.asInstanceOf[js.Array[ResizeObserverBorderBoxSize]](0).inlineSize match {
      case width: Double => println(width)
      case _             => 
    }
  } catch {
    case e => dom.console.debug(s"Resize Observer error: ${e.getMessage}", e)
  }
}
```

**Comment by yurique** [comment]
Interesting! I wonder why I missed the arrays there? 🤔 
Maybe because they say it's an object in the summary? 🤔 

> ResizeObserverEntry.borderBoxSize
> An object containing the new border box size of the observed element when the callback is run.

<img width="809" alt="image" src="https://github.com/tulz-app/laminext/assets/919230/ca902af7-af71-4983-95f7-d391d503fc33">

(they only say it's an array in the detailed field docs)


Thanks for pointing this out! I'll probably release a patch version even though it's a "breaking" change.

**Comment by yurique** [comment]
Also, these types are now available in scala-js-dom, so I'll just use those.

**Comment by yurique** [comment]
Fixed in 0.16.1 (https://github.com/tulz-app/laminext/releases/tag/v0.16.1)

(also added an example https://laminext.dev/v/0.16.x/core/example-resize-observer/live)
